frontend/static/locales/eu.json

```diff
@@ -397,6 +463,31 @@
   "Your TXID": "Zure TXID",
   "Lightning": "Lightning",
   "Onchain": "Onchain",
+  "open_dispute": "Gatazka bat irekitzeko itxaron behar duzu <1><1/>",
```

Comment:
I don't clearly know the context, but the literal translation would be: "Ireki gatazka"